messaging/src/androidTest/java/io/lantern/messaging/metadata/MetadataTest.kt

```diff
@@ -76,24 +76,54 @@ class MetadataTest : BaseTest() {
             assertTrue(peak.toDouble() / average > 100)
             assertEquals("8.853", md.additionalMetadata?.get("duration"))
 
-            // print out the waveform for visual inspection
-            val builder = StringBuilder()
-            for (i in 0..255) {
-                val referenceLevel = 255 - i
-                builder.append("$referenceLevel    ")
-                bars.forEach {
-                    val level = it
-                    if (level >= referenceLevel) {
-                        builder.append('A')
-                    } else {
-                        builder.append(' ')
-                    }
+            printWaveform(bars)
+        }
+    }
+
+    @Test
+    fun testAudioLargeMp3() {
```

Comment:
New test that uses a 3 minute mp3 file (recording of the Ode to Joy). I visually inspected the printed waveform to make sure it looks reasonable given the content.